src/components/feed/CreateFeed/createfeed.scss

```diff
@@ -263,3 +263,4 @@ Styles for pipelines
   }
 
 }
+}
```

Comment:
I see that this same delta was made in #694: https://github.com/FNNDSC/ChRIS_ui/pull/694/files#diff-299e5a7794b1d141af08a435373d3ae31aec77601cfdcc0194c59ad23e834ce2R266

It's concerning to see two closing brackets at the same indentation level. What is happening here?